e2e/docker/regressions.go

```diff
@@ -344,6 +345,9 @@ func (c ctx) issue1670(t *testing.T) {
 			}
 		})
 
+		dirs.MkdirOrFatal(t, filepath.Join(tmpDir, "upper"), 0o777)
```

Comment:
Generally, the need to add additional operations to get an existing test to pass is a good pointer that you need to check, empirically, the behavior of OCI mode vs native mode.

It might actually be necessary, but in those cases it'd also be good to leave a concise justification for reviewers.